src/client.rs

```diff
@@ -14,4 +14,55 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-pub struct Client {}
+use crate::error::{Error, Result};
+use crate::proto::milvus::milvus_service_client::MilvusServiceClient;
+use tonic::codegen::StdError;
+use tonic::transport::Channel;
+
+pub struct Client {
+    client: MilvusServiceClient<Channel>,
+}
+
+impl Client {
+    pub async fn new<D>(dst: D) -> Result<Self>
```

Comment:
Does `tonic::transport::Endpoint` require the protocol prefix like "http://"? If yes, the parameter(s) should be `(addr: &str)`, or `(host: &str, port: i32)`, which hide the unimportant details (prefer the former)

Reply:
or Into<String>